edx-solutions/edx-platform
# [MCKIN-11139] BB-1347 Add polish translations for multiple libraries/xblocks

Add polish translations for api-integration, edx-notifications and xblock-dscussion

**JIRA tickets**: 

**Discussions**: 

**Dependencies**: None

**Screenshots**: Always include screenshots if there is any change to the UI.

**Sandbox URL**: TBD - sandbox is being provisioned.

**Merge deadline**: "None" if there's no rush, "ASAP" if it's critical, or provide a specific date if there is one.

**Testing instructions**:

1. Checkout this branch
2. Change language to polish
3. Check notifications and discussions show translations.

**Reviewers**

**Settings**
```yaml
EDXAPP_FEATURES:
  ENABLE_COMBINED_LOGIN_REGISTRATION: true
```
